src/types/sqs.ts

```diff
@@ -11,6 +11,7 @@ export enum EventName {
   BridgePreProcess = 'BridgePreProcess',
   ContractDeployed = 'ContractDeployed',
   AvailableOperatorJob = 'AvailableOperatorJob',
+  Transfer = 'Transfer',
```

Comment:
Also, following Angello's suggestion I think it's better to use `ERC721Transfer`

Reply:
okay i was trying to keep it consistent with the actual event name, but if this helps filter it's cool with me since we'll eventually have ERC20 transfers too